inc/packages/admin/info.php

```diff
@@ -430,6 +435,76 @@ function add_requirement_notices( ReleaseDocument $release ) : void {
 	}
 }
 
+/**
+ * Render the validation notice.
+ *
+ * Renders the validation status for the package's alias. Also returns a bool
+ * indicating whether the package is "safe" to install - packages which fail
+ * validation are not safe, while those without an alias or with a valid alias
+ * are safe.
+ *
+ * @param DIDDocument $did DID to validate.
+ * @return bool True if the package is "safe" to install, false if install should be blocked.
+ */
+function render_alias_notice( DIDDocument $did ) : bool {
+	$validation = Packages\validate_package_alias( $did );
+	$title = __( 'Domain Alias', 'fair' );
+	$result = false;
+	switch ( gettype( $validation ) ) {
+		case 'string':
+			$message = sprintf(
+				/* translators: %1$s: full URL for validated domain, %2$s: raw domain */
+				__( '<strong>Validated</strong> as <a href="%1$s">%2$s</a>', 'fair' ),
+				esc_url( 'https://' . $validation . '/' ),
+				esc_html( $validation )
+			);
+			$result = true;
+			break;
+
+		case 'NULL':
+			$message = __( 'Not validated: No domain alias is set', 'fair' );
+			$result = true;
+			break;
+
+		default:
+			if ( ! is_wp_error( $validation ) ) {
+				// Invalid type, assume failure.
+				$validation = new WP_Error(
+					'fair.packages.admin.info.validation_notice.invalid_result',
+					__( 'An unknown error occurred', 'fair' )
+				);
+			}
+			$message = sprintf(
+				'<strong>%s</strong>',
+				esc_html__( 'Validation failed', 'fair' )
+			);
+			add_action( 'minifair.render.notices', function () use ( $validation ) {
+				wp_admin_notice(
+					sprintf(
+						/* translators: %s: validation error message */
+						__( '<p><strong>Error:</strong> Failed domain alias validation, this package may be unsafe: %s</p>', 'fair' ),
+						esc_html( $validation->get_error_message() )
+					),
+					[
+						'type'               => 'error',
+						'additional_classes' => [ 'notice-alt' ],
+						'paragraph_wrap'     => false,
+					]
+				);
+			} );
+			$result = false;
+			break;
+	}
+
+	printf(
+		'<strong>%s</strong>: %s',
```

Comment:
Technically, languages like French could have something like <strong>%s</strong> : %s - Note the space after the colon. That's a simple example, but shows we might want that to be translatable too.